### ISSUE TITLE
'Make an API Request' error

### ISSUE THREAD

**Issue by shcherbak** [mulesoft/api-workbench#441]
[Enter steps to reproduce:]

1. open api console
2. press make an api request

**Atom**: 1.15.0 x64
**Electron**: 1.3.13
**OS**: "Fedora release
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package 0.8.44

### Stack Trace

Uncaught TypeError: e.examples is not a function

```
At /home/zip/.atom/packages/api-workbench/main.js:6

TypeError: e.examples is not a function
    at /packages/api-workbench/main.js:6:6272
    at Array.forEach (native)
    at e.render (/packages/api-workbench/main.js:6:6220)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:789:34)
    at ReactCompositeComponentWrapper._renderValidatedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:816:14)
    at ReactCompositeComponentWrapper.mountComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:237:30)
    at Object.mountComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:38:35)
    at ReactDOMComponent._mountChildByNameAtIndex (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:400:40)
    at ReactDOMComponent._updateChildren (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:306:16)
    at ReactDOMComponent.updateChildren (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:251:14)
    at ReactDOMComponent._updateDOMChildren (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:470:12)
    at ReactDOMComponent.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:319:10)
    at ReactDOMComponent.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:303:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at Object.updateChildren (/packages/api-workbench/node_modules/react/lib/ReactChildReconciler.js:83:25)
    at ReactDOMComponent._updateChildren (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:277:47)
    at ReactDOMComponent.updateChildren (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:251:14)
    at ReactDOMComponent._updateDOMChildren (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:470:12)
    at ReactDOMComponent.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:319:10)
    at ReactDOMComponent.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:303:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at Object.updateChildren (/packages/api-workbench/node_modules/react/lib/ReactChildReconciler.js:83:25)
    at ReactDOMComponent._updateChildren (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:277:47)
    at ReactDOMComponent.updateChildren (/packages/api-workbench/node_modules/react/lib/ReactMultiChild.js:251:14)
    at ReactDOMComponent._updateDOMChildren (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:470:12)
    at ReactDOMComponent.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:319:10)
    at ReactDOMComponent.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactDOMComponent.js:303:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper._updateRenderedComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:748:23)
    at ReactCompositeComponentWrapper._performComponentUpdate (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:726:10)
    at ReactCompositeComponentWrapper.updateComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:642:12)
    at ReactCompositeComponentWrapper.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:506:10)
    at Object.receiveComponent (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:97:22)
    at ReactCompositeComponentWrapper.performUpdateIfNecessary (/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:524:23)
    at Object.performUpdateIfNecessary (/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:115:22)
    at runBatchedUpdates (/packages/api-workbench/node_modules/react/lib/ReactUpdates.js:151:21)
    at ReactReconcileTransaction.perform (/packages/api-workbench/node_modules/react/lib/Transaction.js:134:20)
    at ReactUpdatesFlushTransaction.perform (/packages/api-workbench/node_modules/react/lib/Transaction.js:134:20)
    at ReactUpdatesFlushTransaction.perform (/packages/api-workbench/node_modules/react/lib/ReactUpdates.js:95:38)
    at Object.flushBatchedUpdates (/packages/api-workbench/node_modules/react/lib/ReactUpdates.js:175:19)
    at ReactDefaultBatchingStrategyTransaction.closeAll (/packages/api-workbench/node_modules/react/lib/Transaction.js:207:25)
    at ReactDefaultBatchingStrategyTransaction.perform (/packages/api-workbench/node_modules/react/lib/Transaction.js:148:16)
    at Object.batchedUpdates (/packages/api-workbench/node_modules/react/lib/ReactDefaultBatchingStrategy.js:66:19)
    at Object.batchedUpdates (/packages/api-workbench/node_modules/react/lib/ReactUpdates.js:110:20)
    at dispatchEvent (/packages/api-workbench/node_modules/react/lib/ReactEventListener.js:174:20)
```

### Commands

```
  2x -1:43.1.0 api-workbench:console (div.scrollpad.pane-item.padded.raml-details)
```

### Non-Core Packages

```
api-workbench 0.8.44 
busy-signal 1.3.0 
intentions 1.1.2 
linter 2.1.1 
linter-ui-default 1.2.1 
```

**Comment by dreamflyer** [comment]
@shcherbak can not to reproduce. Please provide exact reproduction sequence including raml file on which this issue occurs.

**Comment by shcherbak** [comment]
@dreamflyer , here is the project files
[raml.tar.gz](https://github.com/mulesoft/api-workbench/files/864019/raml.tar.gz)

**Comment by dreamflyer** [comment]
will be published in v0.8.45.
